README.md

```diff
@@ -132,6 +132,24 @@ do this set the `:fail-fast?` option to `true`:
 user=> (run-tests (find-tests "test") {:fail-fast? true})
 ```
 
+#### Long test reporting
+
+If you wish to monitor the length of time to run each test, you can set the
```

Comment:
Can you ensure lines are wrapped at 72 characters in the README?

Reply:
done